dojo/tools/kubehunter/parser.py

```diff
@@ -0,0 +1,99 @@
+import hashlib
+import json
+from dojo.models import Finding
+
+
+class KubeHunterParser(object):
+    """
+    kube-hunter hunts for security weaknesses in Kubernetes clusters. The tool was developed to increase awareness and visibility for security issues in Kubernetes environments.
+    """
+
+    def get_scan_types(self):
+        return ["KubeHunter Scan"]
+
+    def get_label_for_scan_types(self, scan_type):
+        return "KubeHunter Scan"
+
+    def get_description_for_scan_types(self, scan_type):
+        return "KubeHunter JSON vulnerability report format.."
+
+    def get_findings(self, file, test):
+        data = json.load(file)
+
+        dupes = dict()
+
+        # Find any missing attribute
+        vulnerabilities = data['vulnerabilities']
+        check_required_attributes(vulnerabilities)
+
+        for item in vulnerabilities:
+            vulnerability_id = item.get('vid')
+            title = item['vulnerability']
+
+            # Finding details information
+            findingdetail = '**Hunter**: ' + item.get('hunter') + '\n\n'
+            findingdetail += '**Category**: ' + item.get('category') + '\n\n'
+            findingdetail += '**Location**: ' + item.get('location') + '\n\n'
+            findingdetail += '**Description**:\n' + item.get('description') + '\n\n'
+
+            # Finding severity
+            severity = item.get('severity', 'info')
+            allowed_severity = ['info', 'low', 'medium', 'high', "critical"]
+            if severity.lower() in allowed_severity:
+                severity = severity.capitalize()
+            else:
+                severity = 'Info'
+
+            # Finding mitigation and reference
+            avd_reference = item.get('avd_reference')
+
+            if avd_reference and avd_reference != '' and vulnerability_id != 'None':
+                mitigation = f"Further details can be found in kube-hunter documentation available at : {avd_reference}"
+                references = "**Kube-hunter AVD reference**: " + avd_reference
+            else:
+                mitigation = ''
+                references = ''
```

Comment:
If I remember correctly, empty `mitigation` and `references` should be `None`, not `''`